core/lib/spree/event/firing.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Spree
+  module Event
+    # The result of firing an event
+    # 
+    # It encapsulates a fired {Spree::Event::Event} as long as the
+    # {Spree::Event::Execution}s it originated.
+    #
+    # This class is useful mainly for debugging and logging purposes. An
+    # instance of it is returned on {Spree::Event.fire}.
+    class Firing
+      # Fired event
+      #
+      # @return [Spree::Event::Event]
+      attr_reader :event
+
+      # Listener executions that the firing originated
+      #
+      # @return [Array<Spree::Event::Execution>]
+      attr_reader :executions
+
+      # @api private
+      def initialize(event: :event, executions: :executions)
```

Comment:
Not sure I understand the reason behind the default values?

Reply:
The only apparent reason is that my brain was fried at some development point :smile: Thanks for spotting it, fixed!